api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TXTBCohortQueries.java

```diff
@@ -302,42 +302,6 @@ public CohortDefinition artList() {
     addGeneralParameters(cd);
     return cd;
   }
-  /**
-   * BR-12 All patients from denominator who have the following requests/results registered during
-   * the period:
-   *
-   * @return
-   */
-  public CohortDefinition getPatientsPositiveResultsReturnedComposition() {
-    CompositionCohortDefinition cd = new CompositionCohortDefinition();
-    cd.addParameter(new Parameter("startDate", "startDate", Date.class));
-    cd.addParameter(new Parameter("endDate", "endDate", Date.class));
-    cd.addParameter(new Parameter("location", "location", Location.class));
-
-    String mappings = "startDate=${startDate},endDate=${endDate},location=${location}";
-
-    CohortDefinition genexpertTestPositive =
-        genericCohortQueries.generalSql(
-            "genexpert-positivo",
-            TXTBQueries.tbGenexpertTest(
-                hivMetadata.getAdultoSeguimentoEncounterType().getEncounterTypeId(),
-                tbMetadata.getTBGenexpertTestConcept().getConceptId(),
-                commonMetadata.getPositive().getConceptId(),
-                null));
-
-    cd.addSearch("genexpert-positivo", EptsReportUtils.map(genexpertTestPositive, mappings));
-
-    cd.addSearch("result-basiloscopia", EptsReportUtils.map(getResultForBasiloscopia(), mappings));
-
-    cd.addSearch("test-tb-lam", EptsReportUtils.map(getTestTBLAM(), mappings));
-
-    cd.addSearch("culture-test", EptsReportUtils.map(getCultureTest(), mappings));
-
-    cd.setCompositionString(
-        "genexpert-positivo OR result-basiloscopia OR test-tb-lam OR culture-test");
-
-    return cd;
-  }
```

Comment:
@iteliolucas Any reason to remove this?

Reply:
This piece of code is no longer in use, and should have been removed in the latest fix.